src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -520,6 +522,7 @@ extern char MPIDI_NM_strings[][MPIDI_MAX_NETMOD_STRING_LEN];
 
 int MPIDI_NM_mpi_init_hook(int rank, int size, int appnum, int *tag_bits, MPIR_Comm * init_comm);
 int MPIDI_NM_mpi_finalize_hook(void);
+int MPIDI_NM_mpi_post_int(void);
```

Comment:
`post_int` -> `post_init`. Also the `_mpi_` is not appropriate for functions that do not have an `MPI_` parent. I guess it should just be `MPIDI_NM_post_init`.

Reply:
> `post_int -> post_init`

Such blaring mistake! 😄 

> I guess it should just be `MPIDI_NM_post_init`.

I agree.

Reply:
Fixed (actually only this instance, I guess only matters for no-inline build).